egs2/l3das22/enh1/local/metric.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+[ -f ./path.sh ] && . ./path.sh
+. parse_options.sh || exit 1;
+
+pip show -f jiwer >/dev/null || pip install jiwer
+pip show -f pystoi >/dev/null || pip install pystoi
```

Comment:
I think `pystoi` is ensured to be installed in setup.py

Reply:
Thanks! Let me remove it.